corehq/warehouse/management/commands/commit_table.py

```diff
@@ -15,39 +15,40 @@
 
 class Command(BaseCommand):
     """
-    Example: ./manage.py stage_table group_staging -s 2017-05-01 -e 2017-06-01
+    Example: ./manage.py stage_table group_staging 222617b9-8cf0-40a2-8462-7f872e1f1344
     """
     help = USAGE
 
     def add_arguments(self, parser):
         parser.add_argument('slug')
+        parser.add_argument('batch_id')
 
-        parser.add_argument(
-            '-s',
-            '--start_datetime',
-            dest='start',
-            required=True,
-            help='Specifies the last modified datetime at which records should start being included',
-            type=_valid_date
-        )
-        parser.add_argument(
-            '-e',
-            '--end_datetime',
-            dest='end',
-            required=True,
-            help='Specifies the last modified datetime at which records should stop being included',
-            type=_valid_date
-        )
-
-    def handle(self, slug, **options):
-        start = options.get('start')
-        end = options.get('end')
+    def handle(self, slug, batch_id, **options):
+        try:
+            batch_record = BatchRecord.objects.get(batch_id=batch_id)
+        except BatchRecord.DoesNotExist:
+            raise CommandError('Invalid batch ID: {}'.format(batch_id))
 
         try:
             model = get_cls_by_slug(slug)
         except KeyError:
             raise CommandError('{} is not a valid slug. \n\n {}'.format(slug, USAGE))
-        model.commit(start, end)
+
+        commit_record = CommitRecord(
+            slug=slug,
+            batch_record=batch_record,
+            verified=False,
+        )
+        try:
+            commit_record.verified = model.commit(batch_record)
```

Comment:
gio i'm thinking that the `commit` function can return True or False based on whether or not the commit is valid. this is based on the conversation we had some time ago about counting the number of records entered and checking to ensure it was the expected amount. (this functionality has not been implemented)